src/main/export/export.ts

```diff
@@ -1,18 +1,43 @@
 /* eslint-disable no-plusplus */
 import { BrowserWindow } from 'electron';
-import { writeFileSync } from 'fs';
+import * as promises from 'fs/promises';
```

Comment:
Just import as fs - `import fs from 'fs/promises`

Reply:
Just imported the necessary functions `{ access, writeFile }`